samples/interactive-tutorials/src/main/java/events/ImportUserEventsGcs.java

```diff
@@ -34,91 +34,95 @@
 import com.google.cloud.retail.v2.UserEventServiceClient;
 import com.google.longrunning.Operation;
 import com.google.longrunning.OperationsClient;
-import events.setup.EventsCreateGcsBucket;
 import java.io.IOException;
+import java.util.concurrent.TimeUnit;
 
 public class ImportUserEventsGcs {
 
   public static void main(String[] args) throws IOException, InterruptedException {
-    // TODO(developer): Replace these variables before running the sample.
```

Comment:
Leave the comment and use example intalizations everywhere else as well. See https://googlecloudplatform.github.io/samples-style-guide/#run-function

Reply:
Done